.gitignore

```diff
@@ -1,6 +1,7 @@
-vendor
-./composer.lock
-./composer.phar
-data
+vendor/
+composer.lock
+composer.phar/
```

Comment:
I don't think composer.phar is a directory. You can remove the trailing /

Reply:
Yeah, it's an archive which has directories in it. Slash removed.